docs/can-guides/Guides.md

```diff
@@ -1,10 +1,11 @@
 @page guides Guides
 @parent canjs 1
 @group guides/getting-started 0 getting started
-@group guides/experiment 1 experiment
-@group guides/commitment 2 commitment
-@group guides/contribute 3 contribute
-@group guides/upgrade 4 upgrade
+@group guides/experiment 2 experiment
+@group guides/commitment 3 commitment
```

Comment:
I think this should be 4. Right now I’m seeing some weird behavior when switching between pages:
![recipes nav bug](https://cloud.githubusercontent.com/assets/10070176/26610278/77924e46-455b-11e7-9780-d73297597ba8.gif)